src/schema/annotation.yaml

```diff
@@ -50,13 +53,12 @@ classes:
     slot_usage:
       seqid:
         description: The ID of the landmark used to establish the coordinate system for the current feature.
-        range: string ## make this an object
+        range: string
```

Comment:
Question: is this comment requesting for `seqid` to have some other class enforced as its range? A more constrained class perhaps, with a `pattern`?

Reply:
I don't know. You have a good question, but that ## comment predates me. See point about converting YAML # comments into LinkML `notes` above.

Reply:
Awesome, was curious if you knew. But no worries, documenting in a `todo` for now should be good enough.